test/studies/champs/sub_test

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Custom sub_test to run CHAMPS testing. Clones, builds CHAMPS and runs testing.
+
+CHAMPS_BRANCH=${CHAMPS_BRANCH:-development}
+
+CHAMPS_PATCH_PATH=$CHAMPS_SUPPORT_PATH/patches
+CHAMPS_GRAPH_PATH=$CHAMPS_SUPPORT_PATH/graph-infra
+CHAMPS_DATA_PATH=$CHAMPS_SUPPORT_PATH/data
+CHAMPS_CFG_PATH=$CHAMPS_SUPPORT_PATH/configs
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+source $CWD/functions.bash
+
+subtest_start
+
+export CHAMPS_HOME=$CWD/champs
+rm -rf ${CHAMPS_HOME}
+
+# Clone CHAMPS 
+if ! git clone --depth=1 ${CHAMPS_URL} --branch=${CHAMPS_BRANCH} ; then
+  log_fatal_error "cloning CHAMPS"
+else
+  echo "Cloned CHAMPS successfully"
+fi
+cd ${CHAMPS_HOME}
+
+CHAMPS_VERSION=$(git describe --tags --abbrev=0)
+
+# Apply patches
+for p in $CHAMPS_PATCH_PATH/*patch;
+do
+  apply_patch $p
+done
+
+if [ "${CHPL_LLVM}" != "none" ] ; then
+  apply_patch $CHAMPS_PATCH_PATH/optional/llvm.patch
+fi
+
+
+# Compile CHAMPS executables
+test_compile_exec prep
```

Comment:
To me `test_compile_exec` read as "test the compilation and execution", but _think_ this just compiles.